src/Microsoft.ServiceFabric.Services/Runtime/StatefulService.cs

```diff
@@ -8,7 +8,7 @@ namespace Microsoft.ServiceFabric.Services.Runtime
     using Microsoft.ServiceFabric.Data;
 
     /// <summary>
-    /// Represents base class for Microsoft Service Fabric based stateful reliable service
+    /// Represents the base class for Microsoft Service Fabric based on stateful reliable service
```

Comment:
@v-justch  Should just be "Represents the base class for Microsoft Service Fabric based stateful reliable service"